omnigibson/robots/manipulation_robot.py

```diff
@@ -55,10 +55,17 @@ def can_assisted_grasp(obj):
     Returns:
         bool: Whether or not this object can be grasped
     """
-    # Use fallback based on mass
-    mass = obj.mass
-    print(f"Mass for AG: obj: {mass}, max mass: {m.ASSIST_GRASP_MASS_THRESHOLD}, obj: {obj.name}")
-    return mass <= m.ASSIST_GRASP_MASS_THRESHOLD
+
+    # Allow based on mass
+    mass = obj.links[link_name].mass
```

Comment:
was this added so that heavy articulated objects can still be grasped, e.g.: cabinets?